vnext/Microsoft.ReactNative/Views/SwitchViewManager.cpp

```diff
@@ -102,6 +104,19 @@ void SwitchShadowNode::updateProperties(const folly::dynamic &&props) {
   m_updating = false;
 }
 
+void SwitchShadowNode::dispatchCommand(
```

Comment:
Could we fold this into the existing `SwitchViewManager::DispatchCommand` where we implement the ViewManager command instead of implementing on the ShadowNode?

Reply:
I don't think so because the m_updating event suppression data member is on the shadow node, I looked into exposing it to the view manager but couldn't find a way to access the shadow node from the view manager

Reply:
Oh yes, that makes sense. In that case would you be willing to move the code for `updateValue` currently in `SwitchViewManager::DispatchCommand` to `SwitchShadowNode::dispatchCommand`?